ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -119,14 +120,19 @@
   when:
     - _layout.vms.crc is defined
   vars:
-    crc_private_key: "{{ _layout.vms.crc.image_local_dir }}/id_ecdsa"
+    crc_private_key: >-
+      {{
+        (_layout.vms.crc.image_local_dir, "id_ecdsa") |
+        path_join
+      }}
   block:
     - name: Copy authorized_keys
-      ansible.builtin.command:
+      ansible.builtin.shell:
         cmd: >-
```

Comment:
Hard-coding the user here is making my eye twitch. But, I understand that is OK for now

Reply:
In this case I don't think we hit any issue. crc (and OCP actually) have that admin user.